packages/snap-preact/src/Snap.tsx

```diff
@@ -338,6 +337,11 @@ export class Snap {
 							/>,
 							elem
 						);
+
+						if (window?.searchspring) {
+							window.searchspring = undefined;
```

Comment:
Since we are in snap-preact and assume a window, can we just do `delete window.searchspring` and eliminate the if block?

Reply:
I removed the `if` check. Getting an error when using `delete` so we need to set this to `undefined`

Also, the action had a failing test. Tried fixing it but it looks like the HEAD request in `getBundleDetails` never resolves/rejects within the jest test and therefore never reaches the new location of the script append.

Reply:
Yea, that test may need rewritten to actually return some data to allow the rest of the code to execute.

Reply:
We should get this out. I'll try and add this test back in with the `snap-preact` typing branch.